hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/RolloutConfigurationView.java

```diff
@@ -83,4 +86,19 @@ private void init() {
         rootPanel.setContent(vLayout);
         setCompositionRoot(rootPanel);
     }
+
+    @Override
+    protected ProxySystemConfigRollout populateSystemConfig() {
+        ProxySystemConfigRollout configBean = new ProxySystemConfigRollout();
+        configBean.setRolloutApproval(
+                readConfigOption(TenantConfigurationProperties.TenantConfigurationKey.ROLLOUT_APPROVAL_ENABLED));
```

Comment:
maybe remove qualifier for readability?

Reply:
Done. Just calling `TenantConfigurationKey.ROLLOUT_APPROVAL_ENABLED` now